docs/source/json-api/production-setup.rst

```diff
@@ -204,13 +204,24 @@ Users may consider running PostgreSQL backend in a `high availability configurat
 The benefits of this are use-case dependent as this may be more expensive for
 smaller active contract datasets, where re-initializing the cache is cheap and fast.
 
-Finally we recommend using app orchestration utilities which monitor the health of the service
-and perform subsequent operations to ensure availability. These utilities can use the
+Finally we recommend using orchestration systems or load balancers which monitor the health of
+the service and perform subsequent operations to ensure availability. These systems can use the
 `healthcheck endpoints <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`__
 provided by the *HTTP JSON API* server. This can also be tied into supporting arbitrary
 autoscaling implementation to ensure minimum number of *HTTP JSON API* servers on
 failures.
 
+Setup the HTTP JSON API Service to work with Highly Available Participants
+--------------------------------------------------------------------------
+
+In case the participant node itself is configured to be highly available, depending on the setup you might want
```

Comment:
"If the participant node itself is configured to be highly available, you can choose different strategies [to do what?] depending on the setup."

Reply:
Addressed by https://github.com/digital-asset/daml/commit/9fd3f064758f65b04e349a175d183236c1b8371f